ecs-agent/api/resource/ebs_discovery_windows_test.go

```diff
@@ -0,0 +1,87 @@
+//go:build windows && unit
+// +build windows,unit
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package resource
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testVolumeID = "vol-0a1234f340444abcd"
+	deviceName   = "/dev/sdf"
+)
+
+func Test_parseExecutableOutput_WithHappyPath(t *testing.T) {
```

Comment:
nit: Could the naming convention of these tests not include any `_` and instead have the first letter be capitalized? For example `TestParseExecutableOutputWithHappyPath`. Here's also the tests for the windows version of the ENI watcher -> https://github.com/mye956/amazon-ecs-agent/blob/master/agent/eni/watcher/watcher_windows_test.go#L107

Reply:
Done.